calico-vpp-agent/cni/cni_server.go

```diff
@@ -442,6 +499,27 @@ forloop:
 	return nil
 }
 
+func (s *Server) createRedirectToHostRules() (uint32, error) {
+	var missNextIndex int32 = -1
+	index, err := s.vpp.AddClassifyTable(&types.ClassifyTable{
+		Mask:           types.DstThreeTupleMask,
+		NextTableIndex: types.InvalidID,
+		MaxNumEntries:  200000,
```

Comment:
`200k` is probably a bit much here 😄 
we could use something like `2 * len(config.redirects)`